src/main/java/ml/shifu/eval/EvalStep.java

```diff
@@ -65,4 +73,20 @@ public Void process() throws IOException {
         return null;
     }
 
+    private void enhanceEvalData(){
+        for(EvalConfig evalConfig: modelConfig.getEvals()){
+            String enhancePigPath = pathFinder.getScriptPath("scripts/EnhanceColumn.pig");
```

Comment:
EvalStep and InitStep is interface for outside reference, for internal case we should add logic inside of EvalModellProcessor and I think this could better be merged together in EvalScoreUDF to save jobs for performance improvement. We would't like to have extra MR/Pig jobs as we can.